pkg/operator/sync_apiservice_v4_00.go

```diff
@@ -10,50 +11,52 @@ import (
 	appsclientv1 "k8s.io/client-go/kubernetes/typed/apps/v1"
 	coreclientv1 "k8s.io/client-go/kubernetes/typed/core/v1"
 
+	"github.com/openshift/library-go/pkg/operator/events"
 	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
 	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
 	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
 	scsv1 "github.com/openshift/service-ca-operator/pkg/apis/serviceca/v1"
+
+	"github.com/openshift/service-ca-operator/pkg/operator/operatorclient"
 	"github.com/openshift/service-ca-operator/pkg/operator/v4_00_assets"
-	"os"
 )
 
 // syncAPIServiceController_v4_00_to_latest takes care of synchronizing (not upgrading) the thing we're managing.
 // most of the time the sync method will be good for a large span of minor versions
-func syncAPIServiceController_v4_00_to_latest(c serviceCertSignerOperator, operatorConfig *scsv1.ServiceCA) error {
+func syncAPIServiceController_v4_00_to_latest(c serviceCAOperator, eventRecorder events.Recorder, operatorConfig *scsv1.ServiceCA) error {
```

Comment:
Extra event recorder being passed here.

Reply:
what do you mean extra? It's required to pass into the resourceapply functions.

Reply:
https://github.com/openshift/service-ca-operator/pull/16#discussion_r259325393